src/mjolnir/valhalla_build_transit.cc

```diff
@@ -774,10 +775,11 @@ void write_pbf(const Transit& tile, const boost::filesystem::path& transit_tile)
   if (!boost::filesystem::exists(transit_tile.parent_path())) {
     boost::filesystem::create_directories(transit_tile.parent_path());
   }
-  std::fstream stream(transit_tile.string(), std::ios::out | std::ios::trunc | std::ios::binary);
-  if (!tile.SerializeToOstream(&stream)) {
+  auto size = tile.ByteSize();
```

Comment:
there is a bunch like this, i use mem_map as an array that is semi in memory semi on disk because the lite runtime doesnt support IOstreams